tests/Frameworks/Symfony/Version_2_3/app/SymfonyRequirements.php

```diff
@@ -405,22 +405,24 @@ public function __construct()
         $this->addRequirement(
             is_dir(__DIR__.'/../vendor/composer'),
             'Vendor libraries must be installed',
-            'Vendor libraries are missing. Install composer following instructions from <a href="http://getcomposer.org/">http://getcomposer.org/</a>. ' .
+            'Vendor libraries are missing. Install composer following instructions from <a href="http://getcomposer.org/">http://getcomposer.org/</a>. '.
                 'Then run "<strong>php composer.phar install</strong>" to install them.'
         );
 
-        $baseDir = basename(__DIR__);
+        $cacheDir = is_dir(__DIR__.'/../var/cache') ? __DIR__.'/../var/cache' : __DIR__.'/cache';
```

Comment:
Where did all these Symfony updates come from? At first I thought you fixed some `Boolean` -> `bool` by hand but there are things in here that you were unlikely to have written.

Reply:
What happened there :-D
I did run the test_web_symfony_23 locally, maybe that wreaked havoc through the code somehow.